initializr-generator/src/main/resources/templates/GatewayIndex.tmpl

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="bulma.min.css"/>
+	<title>Gateway Homepage</title>
+</head>
+<body>
```

Comment:
Can we add a image in the head of the page? something like this:
https://www.google.com/imgres?imgurl=https%3A%2F%2Fres.infoq.com%2Fpresentations%2Fspring-cloud-gateway%2Fen%2Fslides%2Fsl14-1518709821750.jpg&imgrefurl=https%3A%2F%2Fwww.infoq.com%2Fpresentations%2Fspring-cloud-gateway&docid=JHp8KAAFotD7hM&tbnid=ZE56qk121xqFiM%3A&vet=10ahUKEwi3nrT0tuvaAhUIe7wKHSrEDusQMwg8KAMwAw..i&w=1680&h=945&bih=776&biw=1600&q=spring%20cloud%20gateway&ved=0ahUKEwi3nrT0tuvaAhUIe7wKHSrEDusQMwg8KAMwAw&iact=mrc&uact=8?

Reply:
Styles improved.